data/cncf.yaml

```diff
@@ -4227,6 +4279,11 @@
       check_sets:
         - code
         - community
+    - name: bank-vaults.dev
+      url: https://github.com/bank-vaults/bank-vaults.dev
+      check_sets:
+        - docs
+        - community
```

Comment:
There is already one repository with the `community` check set in this project, please remove this check set from the `bank-vaults.dev` repository.